docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md

```diff
@@ -1,8 +1,8 @@
-## Read-Write Splitting Plugin
+# Read-Write Splitting Plugin
 
 The read-write splitting plugin adds functionality to switch between writer/reader instances via calls to the `Connection#setReadOnly` method. Upon calling `setReadOnly(true)`, the plugin will establish a connection to a reader instance and direct subsequent queries to this instance. Future calls to `setReadOnly` will switch between the established writer and reader connections according to the boolean argument you supply to the `setReadOnly` method.
```

Comment:
NIT: is it worth adding "a new connection to a randomly chosen reader instance"

Reply:
which reader is chosen depends on the reader selection strategy (could be random, could be the reader with the least connections), but I'll update to clarify this